File: c2023/HelloWorld2003/lucky.cpp
#include<stdio.h>
int main()
{
	int year = 0;
	int month = 0;
	int day = 0;

	int sum = 0;
	int mod = 0;

	printf("Lucky Star\n");
	printf("==========\n");

	for (int i = 0; i < 5; i++) //i++ -> i = i+1;
	{
		printf("Please enter birthday dd-mm-yyyy : ");
		scanf_s("%d-%d-%d", &day, &month, &year);

		sum = year + month + day;
		mod = sum % 3;
		if (mod == 0)
		{
			printf("You are not LUCKY\n");
		}
		if (mod == 1)
		{
			printf("You are NORMAL\n");
		}
		if (mod == 2)
		{
			printf("You are  LUCKY\n");
		}
	}

	
	return 0;
}

File: Python 2023/lucky.py

zero = "you are lucky in this year,...."
one = "you are normal in this year,...."
two = "you are unlucky in this year,...."

year = eval(input('Birth year: '))
month = eval(input('Birth month: '))
day = eval(input('Birth day: '))

sum = year + month + day
mod = sum % 3
print("Lucy Star Baydin")
if mod == 0:
    print(zero)
elif mod == 1:
    print(one)
elif mod == 2:
    print(two)
